clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/bitbucket/BitbucketArtifactConfiguration.java

```diff
@@ -53,7 +53,6 @@ BitbucketArtifactProviderProperties bitbucketArtifactProviderProperties() {
   @Autowired
   ArtifactCredentialsRepository artifactCredentialsRepository;
 
-  @Autowired
   OkHttpClient bitbucketOkHttpClient;
```

Comment:
I don't think these fields are needed anymore, right? Across all of the ArtifactConfiguration classes.

Reply:
ah, i see! i'll pull them out.